abdulfataiaka/pluralsight-admin
# #161347321 Add client validation

#### What does this PR do
- Adds client validation for manage course

#### Description of work done
- Validate `author`, `title`, `length` and `category` when adding or editing course

#### How should it be manually tested
- Clone the repository `git clone https://github.com/abdulfataiaka/pluralsight-admin.git`
- Checkout to this branch
- Select a course for edit
- Clear the category field
- Click save to see an error

#### Relevant pivotal tracker story
[161347321](https://www.pivotaltracker.com/story/show/161347321)

#### Screenshots
<img width="1440" alt="screen shot 2018-10-19 at 4 51 38 pm" src="https://user-images.githubusercontent.com/32123313/47229298-4edb2880-d3bf-11e8-916c-f4f091697cf8.png">
<img width="1440" alt="screen shot 2018-10-19 at 4 51 44 pm" src="https://user-images.githubusercontent.com/32123313/47229299-4edb2880-d3bf-11e8-94ce-5b32ac72e179.png">
